modules/user_accounts/ajax/getData.php

```diff
@@ -0,0 +1,141 @@
+<?php
```

Comment:
@driusan I think an ajax call is better here. I think the ```toJSON()`` function should be used if you are retrieving all the data required for the page as it basically rebuilds the page. In this case only a subset of data is retrieved. 

An alternative would be to pass the data via the smarty variable ```$tpl_data``` but I thing we should veer towards the ajax approach of loading data since the template approach requires javascript in the template file which is ugly.